satpy/readers/fy4_base.py

```diff
@@ -240,21 +244,12 @@ def get_area_def(self, key):
         pdict['a_name'] = f'{self.filename_info["observation_type"]}_{res}m'
         pdict['p_id'] = f'FY-4, {res}m'
 
-        pdict['nlines'] = pdict['nlines'] - 1
-        pdict['ncols'] = pdict['ncols'] - 1
-
-        pdict['coff'] = pdict['coff'] - 0.5
-        pdict['loff'] = pdict['loff'] + 1
-
```

Comment:
Am I understanding these changes correctly and that these lines are no longer needed because the changes to the coff/loff accomplish the same goal (but more accurately)?

Reply:
Yep. There's a more precise way so we don't need them any more.